site/class-2-style/maridjuice/index.js

```diff
@@ -0,0 +1,34 @@
+function render(b, c) {
```

Comment:
Almost there, but some of the variables still have cryptic names. I suggest logging stuff and renaming variables!